samplelistener/listener/dispatcher_service.bal

```diff
@@ -2,46 +2,43 @@ import ballerina/http;
 import ballerina/jballerina.java;
 
 service class DispatcherService {
```

Comment:
For requests to the resource method to be dispatched concurrently, both the service class and the method have to be isolated. Shall we mark this as isolated and introduce the required changes?

Reply:
Going forward without the isolated keywords as discussed. Will share a document with the reasons to avoid those.